roles/infra/tasks/inventory_adopt.yml

```diff
@@ -86,6 +86,7 @@
     # Wait for 10 mins for host to become available
     timeout: 600
     host: "{{ node.ip }}"
+    search_regex: OpenSSH
```

Comment:
Please also include the check on `wait_for` gateway in this PR.

Reply:
Done